### PR TITLE
[pt/en] Blog tutorial and minor adjustments

### PR DESCRIPTION
**Pull request by andtxr**
Translation of the blog tutorial part 2 and minor markdown adjustments.

### CODE REVIEW

File: en/tutorials-and-examples/blog/part-two.rst [hunk 1]
```diff
@@ -308,8 +309,8 @@ ArticlesController::
 
 .. note::
 
-    You need to include the FlashComponent in any controller where you will use
-    it. If necessary, include it in your AppController.
+    You need to include the :doc:`/components/flash` component in any controller
```
Comment (markstory) [inline en/tutorials-and-examples/blog/part-two.rst]: This path doesn't exist. I think you want `/controllers/components/flash`.

File: en/tutorials-and-examples/blog/part-two.rst [hunk 2]
```diff
@@ -352,8 +353,9 @@ validation. Everyone hates coding up endless forms and their
 validation routines. CakePHP makes it easier and faster.
 
 To take advantage of the validation features, you'll need to use CakePHP's
-FormHelper in your views. The :php:class:`Cake\\View\\Helper\\FormHelper` is
-available by default to all views at ``$this->Form``.
+:doc:`/helpers/form` helper in your views. The
```
Comment (markstory) [inline en/tutorials-and-examples/blog/part-two.rst]: `/views/helpers/form` instead.

File: en/tutorials-and-examples/blog/part-two.rst [hunk 3]
```diff
@@ -468,12 +470,13 @@ like::
...
 If they haven't passed in an ``$id`` parameter, or the article does not
-exist, we throw a ``NotFoundException`` for the CakePHP ErrorHandler to take care of.
+exist, we throw a ``NotFoundException`` for the CakePHP ErrorHandler to take
+care of.
 
 Next the action checks whether the request is either a POST or a PUT request. If
 it is, then we use the POST data to update our article entity by using the
-'patchEntity' method.  Finally we use the table object to save the entity back
-or kick back and show the user validation errors.
+``patchEntit()`` method.  Finally we use the table object to save the entity
```
Comment (markstory) [inline en/tutorials-and-examples/blog/part-two.rst]: Missing a y.

File: pt/tutorials-and-examples/blog/part-two.rst [hunk 4]
```diff
@@ -1,13 +1,666 @@
...
+Quando se usa um formulário para postar dados, essa informação fica disponível
+em ``$this->request->data``. Você pode usar as funções :php:func:`pr()` ou
+:php:func:`debug()` caso queira verificar esses dados.
+
+Usamos os métodos ``success()`` e ``error()`` do ``FlashComponent`` para definir
+uma mensagem que será armazenada numa variável de sessão. Esses métodos são
+gerados usando os `recursos de métodos mágicos
+<http://php.net/manual/en/language.oop5.overloading.php#object.call>`_ do PHP.
+Mensagens flash serão exibidas na página após um redirecionamento. No layout nós
+temos ``<?= $this->Flash->render() ?>``que exibe a mensagem e limpa a variável
```
Comment (markstory) [inline pt/tutorials-and-examples/blog/part-two.rst]: Need a space after the backticks.

**Comment by andtxr** [comment]
Thanks @markstory :+1:

**Pull request merged**